server/src/pages/not-found/model.js

```diff
@@ -1,5 +1,5 @@
 exports.createModel = function(req) {
   return {
-    title: "Page Not Found"
+    title: "Page Not Found" // todo l10n: notFoundPageTitle
```

Comment:
Sounds good to me. And then just use the title l10n id in `<Localized id="…" /> in the view code.